# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  d6605c04d5d6e35eb9d765e0e1de64e4bc06e304
Misc: Set pref mail.html_sanitize.drop_conditional_css to false.

diff --git a/mailnews/mailnews.js b/mailnews/mailnews.js
--- a/mailnews/mailnews.js
+++ b/mailnews/mailnews.js
@@ -730,9 +730,9 @@ pref("mail.smtpserver.default.try_ssl", 
 // If true, SMTP LOGIN auth and POP3 USER/PASS auth, the last of the methods to try, will use Latin1.
 pref("mail.smtp_login_pop3_user_pass_auth_is_latin1", true);
 
 // Strip CSS conditional rules in received and sent mail
-pref("mail.html_sanitize.drop_conditional_css", true);
+pref("mail.html_sanitize.drop_conditional_css", false);
 
 // For the next 3 prefs, see <http://www.bucksch.org/1/projects/mozilla/16507>
 // TXT->HTML :-) etc. in viewer
 pref("mail.display_glyph", true);
